tests/neg-custom-args/fatal-warnings/i9408b/Conv.scala

```diff
@@ -0,0 +1,8 @@
+package test.conversions
+
+import language.`3.0-migration`
+import scala.language.implicitConversions
+
+object Conv {
+  implicit val implicitLength: String => Int = _.length
+}
```

Comment:
BTW, if the intent is to test separate compilation, we need the suffix `Conv_1.scala` and `Test_2.scala`.

Reply:
I mainly wanted to check how a reference to a definition in a different package will be displayed in a warning message but checking that splitting compilation into multiple phases doesn't break anything might be a good idea as well.